src/configmanager.cpp

```diff
@@ -238,6 +239,7 @@ bool ConfigManager::load()
 	string[URL] = getGlobalString(L, "url", "");
 	string[LOCATION] = getGlobalString(L, "location", "");
 	string[WORLD_TYPE] = getGlobalString(L, "worldType", "pvp");
+	string[BLOCK_LOGIN_TEXT] = getGlobalString(L, "blockLoginText", "pvp");
```

Comment:
I think the default message shouldn't just be "pvp" xD

Reply:
haha I forgot to change it, soon I'll make a commit to change it